Tasks/Common/PowershellHelpers/Helpers.ps1

```diff
@@ -76,7 +79,14 @@ function Invoke-ActionWithRetries
             }
             else
             {
-                throw (Get-VstsLocString -Key ActionTimedOut)
+                if($TimeoutAction)
+                {
+                    $TimeoutAction.Invoke()
```

Comment:
Should you throw here? or take that as input?

Reply:
Removing this logic